Gemfile

```diff
@@ -6,7 +6,7 @@ gem "dry-auto_inject"
 gem "dry-matcher"
 
 # gem "trailblazer-operation", path: "../trailblazer-operation"
-# gem "trailblazer-macro", path: "../trailblazer-macro"
+gem "trailblazer-macro", path: "../trailblazer-macro"
```

Comment:
Actually, we need to release same change in `trb-macro` update `development_dependancy` current's gemspec. We don't bundle gem with `path`.

Reply:
Yep, I'm aware of it. I uncommented this line as a wip, because I kinda expected that we can release new trb-macro. I'll adjust to your suggestions :beer: